content/en/blog/_posts/2022-11-21-devicemanager-ga.md/index.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: blog
+title: 'Kubernetes 1.26: Device Manager graduates to GA'
+date: 2022-11-21 ## TODO: Fix this to assigned date
```

Comment:
Highlighting this TODO

Reply:
The date has to change?

Reply:
Hi @swatisehgal 
You will be contacted by release comms with the publishing date after the article deadline has passed.

Reply:
[1.26 tracker board](https://github.com/orgs/kubernetes/projects/98/views/11?sortedBy%5Bdirection%5D=asc&sortedBy%5BcolumnId%5D=22229692&groupedBy%5BcolumnId%5D=15217749&filterQuery=enhancements-status%3ATracked+blog-status%3A%22Ready+for+Review%22) was updated with the publication date of this blog post (Dec 19, 2022). I have updated this PR to capture that.